src/Parser/GeoJsonParser.js

```diff
@@ -145,21 +157,28 @@ const GeometryToCoordinates = {
         return lines;
     },
     multiPolygon(crsIn, crsOut, coordsIn, filteringExtent, options) {
-        const polygons = [];
-        polygons.type = 'multipolygon';
+        // if (coordsIn.length == 1) {
```

Comment:
Why did you comment this?

Reply:
Error on my part, it wasn't supposed to be commented !